cmake/BuildOptionDefaults.cmake

```diff
@@ -23,12 +23,6 @@ set(NRN_ENABLE_REL_RPATH_DEFAULT OFF)
 set(NRN_DYNAMIC_UNITS_USE_LEGACY_DEFAULT OFF)
 set(NRN_NMODL_CXX_FLAGS_DEFAULT "-O0")
 
-# on cray cross compiling environment, use internal readline
```

Comment:
@pramodk we should check that this is no longer a problem (i.e. front vs compute node)